doc/source/whatsnew/v1.0.0.rst

```diff
@@ -204,6 +204,7 @@ Other enhancements
 - Roundtripping DataFrames with nullable integer or string data types to parquet
   (:meth:`~DataFrame.to_parquet` / :func:`read_parquet`) using the `'pyarrow'` engine
   now preserve those data types with pyarrow >= 1.0.0 (:issue:`20612`).
+- :func:`read_excel` now can read binary Excel (``.xlsb``) files by passing ``engine='pyxlsb'``.
```

Comment:
can you add a reference to the new docs section.

Reply:
can you do this one

Reply:
I'll get to it tomorrow